enctests/testframework/main.py

```diff
@@ -43,13 +43,13 @@
 )
 
 
-VMAF_LIB_DIR = os.getenv(
-    'VMAF_LIB_DIR',
-    f'{os.path.dirname(__file__)}/../.venv/usr/local/lib/x86_64-linux-gnu'
+VMAF_MODEL_DIR = os.getenv(
+    'VMAF_MODEL_DIR',
+    '/usr/share/vmaf'
 )
 
-if not Path(VMAF_LIB_DIR, "model", "vmaf_v0.6.1.json").exists():
-    print(f"WARNING: Cannot find VMAF configuration files at path {VMAF_LIB_DIR}")
+if not Path(VMAF_MODEL_DIR, "vmaf_v0.6.1.json").exists():
+    print(f"WARNING: Cannot find VMAF configuration files at path {VMAF_MODEL_DIR}")
```

Comment:
In other areas we choose to exit when we're missing crucial elements like this. Should we consider doing that here as well?

Reply:
Added an exit.